CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Snyk Security Changelog
 
+
+## [2.11.0]
+### Changed
+- added ai fix feedback support
+
+
```

Comment:
2.10.0 is the next release, so it should go into that.

Reply:
Fixed!